python/lsst/ctrl/execute/slurmPlugin.py

```diff
@@ -76,26 +76,25 @@ def submit(self, platform, platformPkgDir):
                 "The working local scratch directory localScratchDir is %s "
                 % localScratchDir
             )
-
             print("The generated Slurm submit file is %s " % generatedSlurmFile)
 
         cmd = "sbatch --mem %s %s" % (totalMemory, generatedSlurmFile)
 
         auser = self.getUserName()
-        jobname = "".join(["glide_", auser])
+        jobname = f"glide_{auser}"
         if verbose:
             print("The unix user name is %s " % auser)
             print("The Slurm job name for the glidein jobs is %s " % jobname)
             print("The user home directory is %s " % self.getUserHome())
 
         if auto:
+            self.largeGlideinsFromJobPressure(generatedSlurmFile)
```

Comment:
The code organization is not easy to follow.  The largeGlideinsFromJobPressure does it's own submission to slurm, but glideinsFromJobPressure returns a count to be submitted by later code.  It would be nice if they behaved the same.